sqlite/src/build.c

```diff
@@ -397,7 +405,7 @@ int sqlite3UserAuthTable(const char *zTable){
 static Table *sqlite3FindTable_int(
   sqlite3 *db,
```

Comment:
Big changes to this function due to upstream algorithm change, simplification of customizations, and additional optional diagnostic messages.  Please review.